datastream-kafka/src/main/java/com/linkedin/datastream/kafka/EmbeddedKafkaCluster.java

```diff
@@ -114,6 +114,7 @@ public void startup() {
       properties.setProperty("log.flush.interval.messages", String.valueOf(1));
       properties.setProperty("log.cleaner.enable", Boolean.FALSE.toString()); //to save memory
       properties.setProperty("offsets.topic.num.partitions", "1");
+      properties.setProperty("offsets.topic.replication.factor", "1");
```

Comment:
Just to understand correctly, this reduced the number of replicas to be maintained to 1 vs the default (which I think is 3)?
Do you know how much time this saves? Curious to know how you identified this to be causing delay

Reply:
Some configurations are not setting enough brokers for this replication to work. The gain is not enough, but solves unnecessary errors.